.github/ISSUE_TEMPLATE/organization-membership-request.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
We should add a note about this process in one or two places in the other files in the community repo so folks can discover this.

Reply:
yes, definitely.  waiting on for this to be published, then we can link to it.